src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+@use 'sass:list';
+@import './variables';
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.section {
+  padding: 10px;
+  max-width: calc(($width * 7) + 6px);
+}
+
+.calendar {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  $days: mon, tue, wed, thu, fri, sat, sun;
+
+  @each $day in $days {
+    $i: list.index($days, $day);
+    &--start-day-#{$day} .calendar__day:first-child {
+      margin-left: calc(($width * ($i - 1)) + (($i - 1) * 1px));
```

Comment:
The calculation for `margin-left` should include the gap size variable instead of a hardcoded `1px` to maintain consistency and to make it easier to change the gap size in the future.